crates/solvers/src/api/mod.rs

```diff
@@ -28,6 +31,31 @@ impl Api {
     }
 }
 
-async fn solve(_auction: axum::extract::Json<dto::Auction>) -> axum::response::Json<dto::Solution> {
-    axum::response::Json(dto::Solution::trivial())
+async fn solve(
+    state: axum::extract::State<Arc<baseline::Baseline>>,
+    auction: axum::extract::Json<dto::Auction>,
+) -> (
+    axum::http::StatusCode,
+    axum::response::Json<dto::Response<dto::Solution>>,
+) {
+    let auction = match auction.to_domain() {
+        Ok(value) => value,
+        Err(err) => {
```

Comment:
Was not logging an error here intentional?
Would you rather want to log the error in the `driver` instead of here?

Reply:
Good point, will add a log. In general, I need a pass-through to make things more ovservable.

Reply:
Added a log.